CRM/Core/Invoke.php

```diff
@@ -102,7 +102,7 @@ public static function hackMenuRebuild($args) {
         return CRM_Utils_System::redirect();
       }
       else {
-        CRM_Core_Error::fatal('You do not have permission to execute this url');
+        CRM_Core_Error::fatal(ts('You do not have permission to execute this url'));
```

Comment:
If the strings are going to be translated, then I think it's good to do a little copy-edit (more consistent/correct grammar). I'll just toss in a few suggestions.

```suggestion
        CRM_Core_Error::fatal(ts('You do not have permission to execute this URL.'));
```